src/battle_script_commands.c

```diff
@@ -5382,7 +5557,7 @@ static void Cmd_moveend(void)
                     gStatuses3[gBattlerTarget] |= STATUS3_SMACKED_DOWN;
                     gStatuses3[gBattlerTarget] &= ~(STATUS3_MAGNET_RISE | STATUS3_TELEKINESIS | STATUS3_ON_AIR);
                     effect = TRUE;
-                    BattleScriptPush(gBattlescriptCurrInstr + 1);
+                    BattleScriptPush(gBattlescriptCurrInstr);
```

Comment:
Note: `gBattlescriptCurrInstr + 1` is inside the `moveend` instruction, which is almost certainly incorrect because it'll cause the rest of the `moveend` effects to be skipped.

You can test that it doesn't work correctly by having a Pokémon holding a Life Orb use Smack Down against a non-grounded target; the Life Orb damage is skipped.